### PR TITLE
Deprecate clerk

### LINKED ISSUE #629: Deprecate clerk role
Seller and dispute resolver entities have a "clerk" role.
The only role clerk has is to initiate the withdrawal of the entity's funds from the protocol to the entity's treasury address.

This responsibility can be moved to the assistant role, which already performs most of the protocol actions.

** To Do **
- [x] Depreacte "clerk" in the `Seller` and `DisputeResolver` entities. Although it won't be used anymore, it should remain in the struct to preserve the storage layout.
- [x] Change `withdrawFunds` method to allow the assistant to withdraw the funds to the treasury.

### PR DESCRIPTION
**Pull request by zajck**
Closes #629

### CODE REVIEW

File: test/example/ZoraWrapperTest.js
Comment (anajuliabit) [inline test/example/ZoraWrapperTest.js:1]: I suppose this file shouldn't belong to this PR

**Review by anajuliabit — CHANGES_REQUESTED** [review]
Could you also please update docs/task.md?

**Review by anajuliabit — APPROVED** [review]

File: contracts/protocol/facets/OrchestrationHandlerFacet1.sol [hunk 1]
```diff
@@ -982,8 +987,9 @@ contract OrchestrationHandlerFacet1 is PausableBase, SellerBase, OfferBase, Grou
      * - The bundles region of protocol is paused
      * - The exchanges region of protocol is paused
      * - The orchestration region of protocol is paused
-     * - Caller is not the supplied assistant and clerk
+     * - Caller is not the supplied assistant and
```
Comment (mischat) [inline contracts/protocol/facets/OrchestrationHandlerFacet1.sol:990]:
```suggestion
     * - Caller is not the supplied assistant
```

**Review by mischat — APPROVED** [review]
this LGTM ... i have suggested a minor copy change, otherwise, it makes sense, and nice to see no mention of the clerk in the contract, par the deprecated storage slots!

**Review by mischat — CHANGES_REQUESTED** [review]
It looks like one of the unit tests isn't passing right now.

**Comment by mischat** [comment]
@zajck it looks like we have unit tests failing, after merging in main

File: contracts/protocol/facets/OrchestrationHandlerFacet1.sol (hunk 1 above)
Comment (zajck) [reply contracts/protocol/facets/OrchestrationHandlerFacet1.sol]: Removed.

**Review by mischat — APPROVED** [review]
LGTM

**Comment by mischat** [comment]
@anajuliabit ^^ you need to re-review, sorry

**Review by anajuliabit — APPROVED** [review]

**Pull request merged**